### ISSUE TITLE
X25519KeyAgreementKey2019 has wrong property

### ISSUE THREAD

**Issue by brianorwhatever** [w3c/vc-data-integrity#70]
In the [example 15](https://w3c.github.io/vc-data-integrity/#example-key-agreement-property-containing-two-verification-methods) the key property is multibase whereas the suite (which I can't seem to find right now) says this should be base58. If this were X25519KeyAgreementKey2020 the multibase would make sense..

**Comment by msporny** [comment]
Good catch, thank you! Perhaps we should just change this key type to `Multikey` in the example, and use a `0xec` value from the Multicodec table: https://github.com/multiformats/multicodec/blob/master/table.csv#L95 ? 

Another options is to just use an ECDSA key in the example. @dlongley, thoughts?
